src/index.js

```diff
@@ -42,7 +42,8 @@ export default class EdiText extends Component {
   }
 
   _handleEnter = e => {
-    if (e.keyCode === 'Enter' || e.code === 'Enter' || e.keyCode === 27) {
+    if (e.keyCode === 'Enter' || e.code === 'Enter' || e.keyCode === 27 || \
```

Comment:
Hey, there is a syntax error here.

Reply:
Thanks for the PR, btw.

Reply:
Hey @alioguzhan , 
Extremely useful component! Thanks for putting it out! I have made a small attempt to handle an edge case.

Reply:
Thanks for your contribution! I am glad if you like it. Merging it...